src/Spice86.Core/Emulator/InterruptHandlers/Input/Mouse/MouseDriver.cs

```diff
@@ -251,4 +260,28 @@ public class MouseDriver : IMouseDriver {
     }
 
     private record MouseRegisters(ushort Es, ushort Ds, ushort Di, ushort Si, ushort Bp, ushort Sp, ushort Bx, ushort Dx, ushort Cx, ushort Ax);
+    
+    /// <inheritdoc />
+    public SegmentedAddress WriteAssemblyInRam(MemoryAsmWriter memoryAsmWriter) {
```

Comment:
I'm not a big fan of setting all this up in advance, just in case a game decides to register a mouse event handler.
Ideally this is done on `RegisterCallback()`.